src/features/defi/providers/fox-farming/components/FoxFarmingManager/Overview/FoxFarmingEmpty.tsx

```diff
@@ -1,33 +1,76 @@
-import { Button, Skeleton, Stack, Text as CText } from '@chakra-ui/react'
+import { Button, Flex, Skeleton, Stack, Text as CText } from '@chakra-ui/react'
 import type { Asset } from '@shapeshiftoss/asset-service'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
 import { DefiModalContent } from 'features/defi/components/DefiModal/DefiModalContent'
 import { EmptyOverview } from 'features/defi/components/EmptyOverview/EmptyOverview'
+import { useCallback, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
 import { Amount } from 'components/Amount/Amount'
-import { Text } from 'components/Text'
+import { RawText, Text } from 'components/Text'
+import { useModal } from 'hooks/useModal/useModal'
+import { selectAssetById, selectPortfolioFiatBalanceByAssetId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
 
 type FoxFarmingEmptyProps = {
   assets: Asset[]
+  assetId: AssetId
   apy: string | undefined
   onClick?: () => void
   opportunityName: string
 }
 
 export const FoxFarmingEmpty = ({
   assets,
+  assetId,
   apy,
   onClick,
   opportunityName,
 }: FoxFarmingEmptyProps) => {
+  const translate = useTranslate()
+  const asset = useAppSelector(state => selectAssetById(state, assetId))
+  const fiatBalance =
+    useAppSelector(state => selectPortfolioFiatBalanceByAssetId(state, { assetId })) ?? '0'
+  const history = useHistory()
+  const {
+    receive: { open },
+  } = useModal()
+
+  const handleLpDepositClick = useCallback(() => {
+    return history.push(
+      '?type=lp&provider=Uniswap%20V2&chainId=eip155%3A1&assetNamespace=erc20&assetReference=0x470e8de2ebaef52014a47cb5e6af86884947f08c&rewardId=&modal=deposit',
+    )
```

Comment:
We might want to extract this string to a constant.

Reply:
Betcha we did <img width="1377" alt="image" src="https://user-images.githubusercontent.com/17035424/229690893-e0e26df9-2fef-4877-bc22-2b12380f3a4b.png">

Reply:
This is just the contract address, need the full URL to direct them to.

Reply:
Can also reference `lpAssetId` we pass as props